### DISCUSSION TITLE
v4.2.5

### DISCUSSION THREAD

**Discussion by Christoph142** [Christoph142/modern-scroll#217] [General]
fix: trigger area doesn't work (thx Amio for reporting)

<hr /><em>This discussion was created from the release <a href='https://github.com/Christoph142/modern-scroll/releases/tag/v4.2.5'>v4.2.5</a>.</em>

**Comment by BLooRT** [comment]
This is simply the best thing that has ever happened to a web browser.  I love modern scroll SO much.  I appreciate it every night... _especially_ in dark mode.  I can't say enough about it.  It perfects dark mode so well, it hurts!
